WordPress/src/main/java/org/wordpress/android/ui/people/PeopleListFragment.java

```diff
@@ -124,10 +119,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         mActionableEmptyView = rootView.findViewById(R.id.actionable_empty_view);
         mFilteredRecyclerView = rootView.findViewById(R.id.filtered_recycler_view);
-        mFilteredRecyclerView
-                .addItemDecoration(new PeopleItemDecoration(getActivity()));
         mFilteredRecyclerView.setLogT(AppLog.T.PEOPLE);
         mFilteredRecyclerView.setSwipeToRefreshEnabled(false);
+        if (context != null) {
+            mFilteredRecyclerView.addItemDecoration(new DividerItemDecoration(context, DividerItemDecoration.VERTICAL));
```

Comment:
Suggestion (💡): Consider replacing `context` with `mFilteredRecyclerView.getContext()` or even `requireContext()`. This will remove the need to create an extra `Context context = getContext();` variable, then check for its nullability and thus making it simpler. Wdyt? 🤔 

PS: Similar concept is applied to fragment classes like `CategoriesListFragment:L58` and `PrepublishingCategoriesFragment:L107` so using `mFilteredRecyclerView.getContext()` in this case might be for the better as it will end up being consistent with the rest.

Reply:
I can't test this right now, but if this works, I agree that we should apply it. I thought `requireContext` was Kotlin specific, but I should have checked if they added a version with the `@NonNull` annotation or not.

I'll look into this when I get a chance, but if you end up taking over this work, please feel free to apply the suggestion yourself.

Thank you for the review! 🙇